src/samples/backlog/example.ts

```diff
@@ -0,0 +1,66 @@
+import { BacklogItemTemplate, BacklogItemType } from "../../models";
+
+export const exampleBacklogItemTemplate: BacklogItemTemplate = {
+  name: "example",
+  description: "",
+  items: [
+    {
+      name: "As a developer, I have automated linting to enforce code style for the protected branches of the ______ project",
+      type: BacklogItemType.Story,
+      description: "We need linting. Linting is good",
```

Comment:
Strongly agree. However, given that we want this to be picked up by people that are new to the project, we could probably explain that linting speeds up code review time by letting reviewers focus on design decisions, and leaving nitty comments to be handled by the compiler. It also helps to prevent [bike shedding](https://en.wikipedia.org/wiki/Law_of_triviality).

Reply:
Totally agree with all of that - I think the purpose of this is more of an example of how things _could_ be structured. I'm hoping that we'll have some real solid content in the actual playbook rather than here. I just wanted to find an example of one thing that we set up in every project.

If it's ok with you, I'll just leave as is, and then when we start creating template content for the playbook (in your fork), we can make the updates then